### PR TITLE
added `pbservice run-pipeline` command

### PR DESCRIPTION
**Pull request by natechols**
With pbsmrtpipe-like syntax.  I had to modify the preset loader to get this to work.

### CODE REVIEW

**Comment by natechols** [comment]
@mpkocher you'd better take a look at my changes to PipelineTemplatePresetLoader.scala before we pull the trigger on this (no rush)

File: smrt-server-analysis/src/main/scala/com/pacbio/secondary/smrtserver/tools/PbService.scala [hunk 1]
```diff
@@ -537,6 +582,112 @@ class PbService (val sal: AnalysisServiceAccessLayer) extends LazyLogging {
...
+
+  protected def getPipelinePresets(presetXml: File): PipelineTemplatePreset = {
+    if (presetXml != null) {
+      val presetPath = Paths.get(presetXml.getAbsolutePath)
+      PipelineTemplatePresetLoader.loadFrom(presetPath)
+    } else defaultPresets
+  }
+
+  protected object OptionTypes {
+    val FLOAT = "pbsmrtpipe.option_types.float"
```
Comment (mpkocher) [inline smrt-server-analysis/src/main/scala/com/pacbio/secondary/smrtserver/tools/PbService.scala]: Is this enum defined somewhere else?

File: smrt-server-analysis/src/main/scala/com/pacbio/secondary/smrtserver/tools/PbService.scala [hunk 2]
```diff
@@ -217,13 +240,29 @@ class PbService (val sal: AnalysisServiceAccessLayer) extends LazyLogging {
   import AnalysisClientJsonProtocol._
 
   protected val TIMEOUT = 10 seconds
+  private lazy val entryPointsLookup = Map(
+    "PacBio.DataSet.SubreadSet" -> "eid_subread",
```
Comment (mpkocher) [inline smrt-server-analysis/src/main/scala/com/pacbio/secondary/smrtserver/tools/PbService.scala]: This believe this is defined in `com.pacbio.secondary.analysis.datasets.DataSetMetaTypes`

**Comment by mpkocher** [comment]
This looks reasonable. Let merge and iterate.

File: smrt-server-analysis/src/main/scala/com/pacbio/secondary/smrtserver/tools/PbService.scala (hunk 1 above)
Comment (natechols) [inline smrt-server-analysis/src/main/scala/com/pacbio/secondary/smrtserver/tools/PbService.scala]: No, they are used here:

https://github.com/PacificBiosciences/smrtflow/blob/master/smrt-server-link/src/main/scala/com/pacbio/secondary/smrtlink/models/SmrtLinkJsonProtocols.scala#L41

and here:

https://github.com/PacificBiosciences/smrtflow/blob/master/smrt-analysis/src/main/scala/com/pacbio/secondary/analysis/jobs/SecondaryJsonProtocols.scala#L137

So it definitely needs to go somewhere central; how about here?

https://github.com/PacificBiosciences/smrtflow/blob/master/smrt-analysis/src/main/scala/com/pacbio/secondary/analysis/jobs/JobModels.scala

**Pull request merged**